src/main_app/services/copysvg_wrapper/extract_translations.py

```
@@ -35,25 +35,23 @@ def _extract_file_translations(
+    file_name = source_file.name if isinstance(source_file, Path) else str(source_file)
+
     try:
-        result_json: dict[str, Any] = extractor.extract_json(source_file)
+        result: TranslationMapping = extractor.extract(source_file)
+    except CopySVGTranslationError as exc:
+        logger.error(f"CopySVGTranslationError on file:{file_name}.")
+        logger.error(f"Error code: {exc.code}")
+        logger.error(f"Error label: {exc.label}")
+
+        return TranslationMapping(error=exc.code)  # , message=exc.label)
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)